style.css

```diff
@@ -499,3 +575,19 @@ form {
   font-weight: 500;
   margin-right: 200px;
 }
+
```

Comment:
- [ ] Kindly make some alignment to this `title and the paragraph` too, they have to be centered 

- [ ] YOUR DESIGN

<img width="726" alt="fm" src="https://user-images.githubusercontent.com/79058364/155741949-0e631fed-5627-4a60-8d2e-faa622c9e0cd.png">

- [ ] THE FIGMA

<img width="301" alt="MAA" src="https://user-images.githubusercontent.com/79058364/155742197-287a578e-a882-4680-929a-571a28888794.png">

Reply:
resolved 👍